packages/x-date-pickers-pro/src/DesktopDateRangePicker/DesktopDateRangePicker.types.ts

```diff
@@ -44,4 +44,9 @@ export interface DesktopDateRangePickerProps<
    * @default {}
    */
   slotProps?: DesktopDateRangePickerSlotProps<TDate, TEnableAccessibleFieldDOMStructure>;
+  /**
+   * If `true`, the popover or modal will close after submitting the full date.
```

Comment:
If we have a specific definition per component, we know its default behavior.
```suggestion
   * If `true`, the popover will close after submitting the full date.
```
Otherwise, we could go the more abstract route to avoid the need to update the JSDoc when we move range pickers away from the tooltip to dialog/popover. 🤔 

```suggestion
   * If `true`, the Picker will close after submitting the full date.
```